### PR TITLE
Update slash commands documentation

### PR DESCRIPTION
**Pull request by justinegeffen**
Changes for https://github.com/mattermost/docs/pull/3190 which has been closed due to forked repo being deleted.

### CODE REVIEW

File: site/content/integrate/admin-guide/admin-slash-commands/_index.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ You can follow these general guidelines to set up a custom Mattermost slash comm
 4. Set the **Request URL** and **Request Method**. The request URL is the endpoint that Mattermost hits to reach your application, and the request method is either POST or GET and specifies the type of request sent to the request URL.
 5. (Optional) Set the response username and icon the command will post messages as in Mattermost. If not set, the command will use your username and profile picture.
 
+**Important:** Custom command URLs do not support basic authentication as the slash commands provide their own authentication level.
```
Comment (aaronrothschild) [inline site/content/integrate/admin-guide/admin-slash-commands/_index.md:63]: What does this mean @justinegeffen @mickmister

Comment (mickmister) [reply site/content/integrate/admin-guide/admin-slash-commands/_index.md:63]: @aaronrothschild I honestly don't know. We use a secret when we send the request but that's it. Looks like the original PR https://github.com/mattermost/docs/pull/3190 had no dev reviewers so the claim/wording was not vetted for correctness. @justinegeffen Do you know why this specific sentence was taken from the other PR, but none of the other sentences from the PR?

Comment (mickmister) [reply site/content/integrate/admin-guide/admin-slash-commands/_index.md:63]: @justinegeffen Maybe we should close this PR?

**Comment by mickmister** [comment]
@justinegeffen I'm not sure why we need to specify that slash commands don't provide [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication), which uses a username/password combination instead of a secret. I wouldn't expect slash commands to use basic authentication, so I'm closing this PR. I can reopen if you think we should.

Thanks for carrying this over and not letting it fall through the cracks 🙂

**Pull request closed without merging**